From 29981acf1f231f99e0ee17ab9a74bdcde047cb46 Mon Sep 17 00:00:00 2001
From: Yufei Gu <yufei.gu@cloudera.com>
Date: Tue, 9 Aug 2016 18:01:43 -0700
Subject: [PATCH 1701/2748] Revert "exclude test
 org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.TestContinuousScheduling"

This reverts commit 54eaa918f435d3d74796bae2476584ad874d9c7f.

Change-Id: Ia3f62cbe3f3e82492e2915e29bdaabef4fbe6bb0
---
 cloudera/excludes.txt |    1 -
 1 file changed, 1 deletion(-)

diff --git a/cloudera/excludes.txt b/cloudera/excludes.txt
index df0816d..82b8c29 100644
--- a/cloudera/excludes.txt
+++ b/cloudera/excludes.txt
@@ -24,4 +24,3 @@ org/apache/hadoop/yarn/server/resourcemanager/metrics/TestSystemMetricsPublisher
 org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestContainerAllocation.java
 org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebApp.java
 org/apache/hadoop/yarn/server/timeline/webapp/TestTimelineWebServices.java
-org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/TestContinuousScheduling.java
-- 
1.7.9.5

